docs/map_state/embeddings.md

```diff
@@ -0,0 +1,24 @@
+Atlas stores, manages and generates embeddings for your unstructured data.
+
+You can access Atlas embeddings in their ambient form (e.g. high dimensional) or in their two-dimensional
+projected representations.
+
+
+```python
+from nomic import AtlasProject
+
+project = AtlasProject(name='My Project')
+
+map = project.maps[0]
+
+map.embeddings.df
```

Comment:
My first natural question here is what columns can we expect? Will we get both the ambient and 2d representations in the same dataframe?